portal/templates/flask_user/emails/eproms_base_message.html

```diff
@@ -6,3 +6,6 @@
 <p>Thank you,</p>
 <p>The TrueNTH Team</p>
 {% endtrans %}
+{% trans %}
```

Comment:
I don't think `parent_org` is populated into *these* email templates, only a `User` object (`user`).

To get something similar, you might have to look at `user.organizations[0]` or [`user.first_top_organization()`](https://github.com/uwcirg/truenth-portal/blob/develop/portal/models/user.py#L785-L797) (haven't tested the below code)

...might also cause exceptions for test users (without orgs)
```suggestion
{% trans parent_org=user.first_top_organization() %}
```

Reply:
Thanks @ivan-c  - I'm going to try the user.first_top_organization() approach.
Based on what I see at https://eproms-dev.cirg.washington.edu/api/user/1465/organization, only one org is at user.organization (indexed at 0), and it's the user's org, not the top parent of that org.